api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -351,41 +360,30 @@ public void testMe_userBeforeSuccessCloudProjectConflict() throws Exception {
     createUser();
     when(fireCloudService.isRequesterEnabledInFirecloud()).thenReturn(true);
 
-    String projectName = BILLING_PROJECT_PREFIX + user.getUserId();
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName);
-    doThrow(new ApiException(HttpStatus.CONFLICT.value(), "conflict"))
-        .when(fireCloudService).createAllOfUsBillingProject(projectName + "-1");
+    ApiException conflict = new ApiException(HttpStatus.CONFLICT.value(), "conflict");
+    doThrow(conflict)
+        .doThrow(conflict)
+        .doNothing()
+        .when(fireCloudService).createAllOfUsBillingProject(anyString());
```

Comment:
This appears to remove the testing on the retry? Am I reading that correctly?

Reply:
Got it. I’d been worried about it because from reading it I thought this was intended to conflict twice then pass. If it’s fine than LGTM.

Reply:
Sorry, that is what this does - I think I slightly mispoke.

- throw conflict
- throw conflict
- return nothing

That was also the previous behavior